### PR TITLE
Add Ali DNS provider

### PR DESCRIPTION
**Pull request by colachg**
Signed-off-by: cola <colachg@gmail.com>

### CODE REVIEW

**Comment by colachg** [comment]
I have tested it myself, it looks good.

**Comment by colachg** [comment]
I think if this was merged, we can close #47 , because I fix something of that and have tested it.

File: alidns/alidns.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Package dnspod adapts the lego Ali DNS
```
Comment (mholt) [inline alidns/alidns.go]: Package alidns

**Review by mholt — CHANGES_REQUESTED** [review]
Thanks! Just a minor nit before merging.

File: alidns/alidns.go (hunk 1 above)
Comment (colachg) [reply alidns/alidns.go]: Em... sorry for my carelessness. :(

**Review by mholt — APPROVED** [review]

**Pull request merged**